tests/no_sir_test.py

```diff
@@ -0,0 +1,11 @@
+import os
+import no_sir
+from errbot.backends.test import testbot
+
+
+class Test_no_sir(object):
+    extra_plugin_dir = '.'
+
+    def test_no_sir(self, testbot):
+        testbot.push_message('sir')
+        assert 'Do not use sir in your conversation' in testbot.pop_message()
```

Comment:
The assert statement does not match the bot output message, the test would always throw an assertion error because of the missing `emot` at the end

Reply:
@sladyn98  it's `in`

Reply:
@meetmangukiya  Yeah your right,so it would pass, but it would be better to keep the outputs same although it doesnt matter :)